[2023-06-11T06:10:36.134+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_pythonoperator_16.get_name manual__2023-06-11T06:10:04.918705+00:00 [queued]>
[2023-06-11T06:10:36.600+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_pythonoperator_16.get_name manual__2023-06-11T06:10:04.918705+00:00 [queued]>
[2023-06-11T06:10:36.600+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-06-11T06:10:37.038+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): get_name> on 2023-06-11 06:10:04.918705+00:00
[2023-06-11T06:10:37.249+0000] {standard_task_runner.py:57} INFO - Started process 226 to run task
[2023-06-11T06:10:37.285+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_pythonoperator_16', 'get_name', 'manual__2023-06-11T06:10:04.918705+00:00', '--job-id', '592', '--raw', '--subdir', 'DAGS_FOLDER/python_operator_dag.py', '--cfg-path', '/tmp/tmp904ppdz8']
[2023-06-11T06:10:37.320+0000] {standard_task_runner.py:85} INFO - Job 592: Subtask get_name
[2023-06-11T06:10:38.038+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_pythonoperator_16.get_name manual__2023-06-11T06:10:04.918705+00:00 [running]> on host 69e374580eab
[2023-06-11T06:10:41.462+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Raja' AIRFLOW_CTX_DAG_ID='dag_pythonoperator_16' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2023-06-11T06:10:04.918705+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-11T06:10:04.918705+00:00'
[2023-06-11T06:10:41.743+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-06-11T06:10:41.896+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dag_pythonoperator_16, task_id=get_name, execution_date=20230611T061004, start_date=20230611T061036, end_date=20230611T061041
[2023-06-11T06:10:42.813+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-11T06:10:43.376+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
